test/test_rigidbody.cpp

```diff
@@ -1341,6 +1342,36 @@ TEST(Joints, combineExtForceAndSoftContact)
     }
 }
 
+TEST(Joints, massMatrixInverse)
+{
+    {
+        Model model(modelPathForGeneralTesting);
+        rigidbody::GeneralizedCoordinates Q(model);
+        FILL_VECTOR(Q, std::vector<double>({-2.01, -3.01, -3.01, 0.1, 0.2, 0.3,
+                                           -2.01, -3.01, -3.01, 0.1, 0.2, 0.3, 0.4}));
+
+        utils::Matrix M(model.massMatrix(Q));
+
+#ifdef BIORBD_USE_CASADI_MATH
+        utils::Matrix Minv_num = M.inverse();
+#else
+        utils::Matrix Minv_num = M.inverse();
+#endif
```

Comment:
If they use the same terminology, it is useless to put them in a separate macro

Reply:
Done.